src/Schema/Events/Click.ts

```diff
@@ -1629,7 +1629,7 @@ export interface ClickedOnDuplicateArtwork {
 export interface ClickedValidationAddressOptions {
   action: ActionType.clickedValidationAddressOptions
   context_module: ContextModule
-  context_page_owner_type: string
```

Comment:
@mdole - might want to peek at force, eigen, etc, to see if this change surfaces a lot of errors from people passing random or invalid strings.

Reply:
yep thanks for the nudge! doing so today